poetry.lock

```diff
@@ -55,6 +55,14 @@ doc = ["packaging", "sphinx-autodoc-typehints (>=1.2.0)", "sphinx-rtd-theme"]
 test = ["contextlib2", "coverage[toml] (>=4.5)", "hypothesis (>=4.0)", "mock (>=4)", "pytest (>=7.0)", "pytest-mock (>=3.6.1)", "trustme", "uvloop (<0.15)", "uvloop (>=0.15)"]
 trio = ["trio (>=0.16)"]
 
+[[package]]
```

Comment:
Please review the `poetry.lock` changes carefully. I'm new to Poetry, and I may have added things in the wrong way (e.g. optional vs required, development vs production requirements). If you find any issues, feel free to fix them directly or point me in the right direction and I'll fix it.